e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/displayissue/DisplayIssueEMTest.java

```diff
@@ -0,0 +1,144 @@
+package org.evomaster.e2etests.spring.examples.displayissue;
+
+import com.foo.rest.examples.spring.bodytypes.BodyTypesController;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.core.search.service.Statistics;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * This class is to test the issue: https://github.com/EMResearch/EvoMaster/issues/765
+ *
+ * In which number of potential faults and number of test targets covered are shown for whitebox
+ * but not for blackbox.
+ *
+ * Run the application
+ *
+ */
+
+public class DisplayIssueEMTest extends SpringTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+
+        // re-using the already developed application BodyTypes
+        BodyTypesController controller = new BodyTypesController();
+        SpringTestBase.initClass(controller);
+    }
+
+    private String findValueOfItemWithKeyInStats(Solution s, String k) {
```

Comment:
generic, reusable functions like this should rather be in `EnterpriseTestBase`, with some JavaDoc

Reply:
Thank you so much Andrea for your comment. As requested, I removed it to EnterpriseTestBase with some JavaDoc.